docs/en/stores/basic-usage.md

```diff
@@ -0,0 +1,471 @@
+# Basic Usage
+
+Dojo is a reactive architecture concerned with constantly modifying and rendering the current state of an application. In simple systems this can happen at a local level and widgets can maintain their own state. However, as a system becomes more complex the need to better compartmentalize and encapsulate data and create a clean separation of concerns quickly grows.
+
+Stores provide a clean interface for storing, modifying, and retrieving data from a global object using a unidirectional data flow. Common patterns such as asynchronous retrieval, middleware, and undo patterns are built in. This allows widgets to remain focused on their primary roles of providing a visual representation of themselves and listening for user interactions.
+
+## The Store
+
+The store hold the global, atomic state for the entire application. It is created when the application is created and defined in the `Registry` with an injector.
+
+> main.ts
+
+```ts
+import Registry from '@dojo/framework/widget-core/Registry';
+import Store from '@dojo/framework/stores/Store';
+import { State } from './interfaces';
+
+const registry = new Registry();
+const store = new Store<State>();
+registry.defineInjector('state', (invalidator) => {
+	store.on('invalidate', () => invalidator());
+	return () => store;
+});
+```
+
+`State` defines the shape of the global store using an interface. In general state should be serializable. This improves performance and makes it easier for Dojo to determine when changes to the data occur.
+
+> interfaces.d.ts
+
+```ts
+interface User {
+	id: string;
+	name: string;
+}
+
+export interface State {
+	auth: {
+		token: string;
+	};
+	users: {
+		current: User;
+		list: User[];
+	};
+}
+```
+
+The above is a simple example for illustrative purposes. Examples in this reference guide will refer back to this interface to reduce repetition.
+
+## Updating stores
+
+To work with Dojo Stores there are three core concepts.
+
+-   **Operations** are instructions to manipulate the state held by the store
+-   **Commands** are simple functions that perform business logic and return operations
+-   **Processes** execute a group of commands and represent application behavior
+
+### Operations
```

Comment:
For this section might be easier to group commands and operations together, that way you will be able to show examples with the recommended approach (i.e. using commands to return operations)

Reply:
I think that's a good idea. If I introduce commands and operations together I won't need to use `store.apply` to show how operations work.